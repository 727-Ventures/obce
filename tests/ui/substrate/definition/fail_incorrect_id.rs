#[obce::definition(id = false)]
pub trait Trait {}

fn main() {}
