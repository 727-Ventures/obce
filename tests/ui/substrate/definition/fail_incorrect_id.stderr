error: id should be integer or string
 --> tests/ui/substrate/definition/fail_incorrect_id.rs:1:20
  |
1 | #[obce::definition(id = false)]
  |                    ^^^^^^^^^^
